landing-pages/site/layouts/meetups/list.html

```diff
@@ -65,6 +65,19 @@ <h5 class="meetups-page page-subtitle">
     </div>
     <div>
         <h4 class="page-header host-header">Want to host a meetup?</h4>
+        <div>
+            Meetups related to the Apache Airflow project are held across the globe, thanks to community volunteers. Interested individuals are encouraged to host community meetups using the name Apache Airflow Meetup <geographical location> in compliance with Apache Software Foundation's branding and trademarks guidelines. No explicit PMC approval is required but also, PMC does not endorse the organizers or communities that organize it.
+            <p/><p/>
+            Hosts are required to ensure that:
+
+            <ul>
+                <li>The Apache Airflow ecosystem should be championed in every meetup and technical session</li>
+                <li>All talks should be vendor-neutral and not sales pitches</li>
+                <li>Each meetup should strive to have at least two talks with speakers representing different companies/organizations if possible to foster neutrality</li>
+                <li>Planned meetups should be brought to the attention of the Airflow dev emailing list</li>
```

Comment:
how about using “mailing list” instead of “emailing list”?
since “mailing list” is the generally used ;)